plugin/api/kotlinx-benchmark-plugin.api

```diff
@@ -34,11 +34,12 @@ public class kotlinx/benchmark/gradle/BenchmarkTarget {
 	public final fun setWorkingDir (Ljava/lang/String;)V
 }
 
-public class kotlinx/benchmark/gradle/BenchmarksExtension {
+public abstract class kotlinx/benchmark/gradle/BenchmarksExtension {
 	public final fun configurations (Lgroovy/lang/Closure;)Lorg/gradle/api/NamedDomainObjectContainer;
 	public final fun getBenchsDescriptionDir ()Ljava/lang/String;
 	public final fun getBuildDir ()Ljava/lang/String;
 	public final fun getConfigurations ()Lorg/gradle/api/NamedDomainObjectContainer;
+	public abstract fun getKotlinCompilerVersion ()Lorg/gradle/api/provider/Property;
```

Comment:
@adam-enko Could you please clarify if this property is intended to be configured by kotlinx-benchmark users?

Reply:
Typically users shouldn't need to configure it, but I left it as public in case they need to update the Benchmarks plugin but keep using an older `kotlin-compiler-embeddable`.

Reply:
> In case they need to update the Benchmarks plugin but keep using an older `kotlin-compiler-embeddable`.

Do you mean in other parts of there benchmark module?

Reply:
I mean in the subproject that has the kotlinx-benchmark Gradle plugin.